virtcontainers/kata_agent.go

```diff
@@ -669,7 +669,6 @@ func constraintGRPCSpec(grpcSpec *grpc.Spec) {
 	// By now only CPU constraints are supported
```

Comment:
The comment is outdated even before this PR.

Reply:
good catch! thanks